pkg/cli/admin/upgrade/status/status.go

```diff
@@ -40,12 +44,23 @@ func New(f kcmdutil.Factory, streams genericiooptions.IOStreams) *cobra.Command
 		},
 	}
 
+	flags := cmd.Flags()
+	// TODO: We can remove these flags once the idea about `oc adm upgrade status` stabilizes and the command
+	//       is promoted out of the OC_ENABLE_CMD_UPGRADE_STATUS feature gate
+	flags.StringVar(&o.mockCvPath, "mock-clusterversion", "", "Path to a YAML ClusterVersion object to use for testing (will be removed later).")
+	flags.StringVar(&o.mockOperatorsPath, "mock-clusteroperators", "", "Path to a YAML ClusterOperatorList to use for testing (will be removed later).")
```

Comment:
Did we want to add some fixtures to this to unit test known configs vs. expected output?  Because running `go test ./pkg/cli/admin/upgrade/status` and having it consume mock data other folks have written and confirm that I haven't broken the output other folks were expecting in those situations would be nice.  Mainly to start building a corpus of situations we want to cover, and as a way to track how our output in those situations evolves over time.

Also, no worries if you think those fixtures sound like overkill, or if they sound useful in the future, but not worth picking up yet.  I'm also fine with a middle ground where we start a directory like `pkg/cli/admin/upgrade/status/fixtures/happy-mid-update`, etc. with the mock  content in it, even if we don't version-control expected output yet.

Reply:
That sounds great, will do!